pkg/cfn/builder/outputs.go

```diff
@@ -141,7 +141,7 @@ func (r *resourceSet) GetAllOutputs(stack cfn.Stack, obj interface{}) error {
 			return errors.Wrap(err, "processing stack outputs")
 		}
 	}
-	logger.Debug("obj = %#v", obj)
+	//logger.Debug("obj = %#v", obj)
```

Comment:
Should we just remove these commented-out lines?

Reply:
Yes, good idea. I just thought these maybe useful another time, but these are rather trivial, and some pre-date unit tests.